packages/node_modules/@webex/helper-image/test/unit/spec/index.js

```diff
@@ -4,11 +4,34 @@
 
 import {assert} from '@webex/test-helper-chai';
 import {readExifData, orient} from '@webex/helper-image';
-import file from '@webex/test-helper-file';
+import fileHelper from '@webex/test-helper-file';
 import sinon from 'sinon';
+import {browserOnly, nodeOnly} from '@webex/test-helper-mocha';
 
 describe('helper-image', () => {
   describe('readExifData()', () => {
+    let buffer;
+
+    browserOnly(before)(() => fileHelper.fetch('/Portrait_7.jpg')
```

Comment:
after fixing returning from the tests, the browser tests were failing because `fileHelper.fetch` in browser fetches the blob instead of a buffer. added this as a workaround to fix the tests
ideally test-helper-file should probably be refactored, but out of scope for this PR